Bottom: 09cd1a18eb13f0784bcf924c068c90413a63a619
Top:    dc16d4b4853b440bcc681781aaf6f3a4493bcb6d
Author: Peter Clifton <pcjc2@cam.ac.uk>
Date:   2009-10-21 00:42:56 +0100

Refresh of add-infrastructure-for-subcomp

---

diff --git a/src/draw.c b/src/draw.c
index b75989a..ea3c301 100644
--- a/src/draw.c
+++ b/src/draw.c
@@ -512,7 +512,7 @@ DrawEverything (BoxTypePtr drawn_area)
       SWAP_IDENT = 0;
       DrawMask (drawn_area);
       SWAP_IDENT = save_swap;
-      gui->set_layer (NULL, SL (FINISHED, 0), 0);
+      gui->set_layer (NULL, SL (FINISHED, 0), 0);
     }
   if (gui->set_layer ("soldermask", SL (MASK, BOTTOM), 0))
     {
@@ -520,7 +520,7 @@ DrawEverything (BoxTypePtr drawn_area)
       SWAP_IDENT = 1;
       DrawMask (drawn_area);
       SWAP_IDENT = save_swap;
-      gui->set_layer (NULL, SL (FINISHED, 0), 0);
+      gui->set_layer (NULL, SL (FINISHED, 0), 0);
     }
   /* Draw top silkscreen */
   if (gui->set_layer ("topsilk", SL (SILK, TOP), 0))
